Dockerfile.tests

```diff
@@ -0,0 +1,20 @@
+FROM registry.centos.org/centos/centos:7
```

Comment:
In `worker`, we build the test image on top of the latest worker image: https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.tests#L1.

Reply:
Currently gemini server mixes both runtime and test dependencies in a single `requirements.txt` file. We could create a separate `requirements.txt` just for the tests dependencies, ideally under `tests/` directory.

You could then follow what we are doing in worker: https://github.com/fabric8-analytics/fabric8-analytics-worker/blob/master/Dockerfile.tests#L5-L6
I.e. copy sources and install the test dependencies.

Reply:
@msrb , but gemini image is not yet on red hat registry, I guess. What would I have to do to get it on RedHat registry

Reply:
Can we do something without it?

Reply:
That should be fine, you actually don't want to download the image from the registry. You want to build it in CI and immediately use it as a base image for your test image. So in your case you'd say something like:
```Dockerfile
FROM registry.devshift.net/fabric8-analytics/fabric8-gemini-server
```

Reply:
But I am actually questioning whether this is a good approach or not. It seems to be overcomplicated and since the entrypoint of this test image will be `pytest`, we won't even test that the base image runs/works as expected. I'd probably advise not to go down this road. cc @tisnik (?)

Reply:
@msrb, So currently tests are passing with host as `0.0.0.0`. Is this acceptable?

Reply:
@GeetikaBatra yeah, that should be fine. I'd maybe use `localhost` instead of `0.0.0.0`, but that would be just my personal preference.

Reply:
@msrb, I will then change it to localhost, and remove the docker files for now. Maybe we can come back to it in future. :)